mantidproject/mantid
# Rounding errors in tables

If a user makes a table with a large number in it. Then opens the table to look at the data (it will look ok), all of the data would have been rounded. This is to stop rounding errors. 

**To test:**
In the script window run the following code:

```
tab = CreateEmptyTableWorkspace()
tab.addColumn('double', 'Asymmetry')
tab.addColumn('double', 'Phase')
tab.addColumn('double', 'DeadTime')
for i in range(0,16):
   #tab.addRow([1, 50.0, 0.00, 0])
   tab.addRow([123456789, 50.0, 1.57])
```
If you then open the table by double clicking it, it will look ok. Close the table and reopen it (on the release branch there will be some rounding of the values). 

You can check the value in the table (without opening it) by using this script:
```

table = mtd["tab"]
print table.row(0)
```



Fixes #20661. 

**Release Notes** 

Added to release notes
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
